src/pages/docs/administration/octopus-ai-assistant/deployment-failure-analyzer.md

```diff
@@ -0,0 +1,72 @@
+
+## How the deployment failure analyzer works
+
+The Deployment Failure Analyzer captures detailed information about deployments, including:
+
+- Deployment logs and error messages
+- Deployment process configuration
+- Script content from deployment steps
+- Build information and artifacts
+- Environment and target details
+
+This context is analyzed by the Octopus AI Assistant to identify the root cause of the failure and provide specific suggestions for resolution.
+
+## Using the deployment failure analyzer
+
+When a deployment fails, you can launch the Octopus AI Assistant from the deployment page. The analyzer will present a suggested prompt for analyzing the failed deployment:
+
+```text
+Help me understand why the deployment failed. If the deployment didn't fail, say so. Provide suggestions for resolving the issue.
+```
+
+The Octopus AI Assistant will analyze the deployment context and provide:
+
+1. **Reason for failure** - The specific step and error that caused the deployment to fail
+2. **What happened** - A detailed breakdown of the deployment process and where it went wrong
+3. **Suggestions for resolving the issue** - Actionable remediation steps with specific commands and configuration changes
+4. **Next steps** - Recommended actions to investigate further and prevent future failures
+
+## Example analysis
+
+Below is a basic example of how the Deployment Failure Analyzer works in practice. The analyzer identified that an Azure Resource Group could not be found during deployment and provided troubleshooting guidance, including verifying the resource group exists, checking Azure account permissions, looking for typos in the configuration, and enabling step retries for intermittent issues.
+
+![Deployment failure analysis example](/docs/administration/octopus-ai-assistant/deployment-failure-analyzer-example.png)
+
+## Adding business logic using custom prompts
+
+For organizations with specific internal processes and troubleshooting procedures, you can enhance the Deployment Failure Analyzer with custom business logic using [custom prompts](/docs/administration/octopus-ai-assistant/custom-prompts). Custom prompts are defined as variables in Library Variable Sets within Octopus Deploy, allowing you to embed organization-specific guidance and next steps directly into the failure analysis responses.
+
+Custom prompts work by combining a user-facing prompt (`PageName[#].Prompt`) with an optional system prompt (`PageName[#].SystemPrompt`) that contains your business logic. The `.Prompt` variable defines what users see and interact with, while the `.SystemPrompt` variable provides behind-the-scenes instructions that guide the AI's analysis without being visible to users.
+
+### Example: Missing Azure resource group
```

Comment:
A missing Azure Resource Group is a basic example. If we wanted a different example, I'm happy, I just didn't have one on-hand I could easily create some prompts and screenshots around